documentation/options.md

```diff
@@ -0,0 +1,42 @@
+Plugin options
+===============
+
+The plugin allows you to specify several options. Here are the default values:
+
+```JavaScript
+new RailsManifestPlugin({
+  fileName: 'manifest.json',
+  writeToFileEmit: false,
+  extraneous: null,
+  mapAssetPath: (requirePath) => requirePath
+});
+```
+
+### fileName
+
+This is the output file that will be created after each change. The default value is `manifest.json`.
+
+### writeToFileEmit
+
+When using this plugin with [webpack-dev-plugin](https://github.com/webpack/webpack-dev-server) the manifest file will not be written to a file, but to an in-memory filesystem used by webpack-dev-plugin.
```

Comment:
This statement doesn't say what it means if the property is set to true or false. Although it can be figured out from the property name, it's not 100% clear.